src/app_engine/constants.py

```diff
@@ -11,9 +11,9 @@
 
 LOOPBACK_CLIENT_ID = 'LOOPBACK_CLIENT_ID'
 
-TURN_BASE_URL = 'https://computeengineondemand.appspot.com'
-TURN_URL_TEMPLATE = '%s/turn?username=%s&key=%s'
-CEOD_KEY = '4080218913'
+TURN_BASE_URL = 'https://networktraversal.googleapis.com'
```

Comment:
I think having a single TURN_URL = 'https://networktraversal.googleapis.com/v1alpha/iceconfig?key=' + os.environ would be simpler than having these 3 variables.